sal_interfaces/MTAOS/MTAOS_Commands.xml

```diff
@@ -6,6 +6,11 @@
     <EFDB_Topic>MTAOS_command_resetCorrection</EFDB_Topic>
     <Description>Reset the current wavefront error correction. The aggregated degree of freedom in optical feedback control will be reset to the initial state.</Description>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>MTAOS</Subsystem>
+    <EFDB_Topic>MTAOS_command_interruptWEP</EFDB_Topic>
+    <Description>Interrupt wep process.</Description>
```

Comment:
Please give more details for this new command. Right now, the `Description` does not give more details than the `EFDB_Topic` itself. Thanks!